Update README.md (#342)
mavlink/MAVSDK-Python: README.md

### Build and install the package locally

> After generating the wrapper and only in ARM architectures with linux, defines a variable `MAVSDK_SERVER_ARCH`:

Reviewer: What about?

```suggestion
If using an ARM architecture on Linux, set the environment variable `MAVSDK_SERVER_ARCH`:
```

```
export MAVSDK_SERVER_ARCH=<ARM embedded architecture>
```

> Supported architectures: armv6l, armv7l and aarch64. For example in Raspberry PI it is armv7l.

Reviewer: ```suggestion
Supported architectures: `armv6l`, `armv7l` and `aarch64`. For example for Raspberry Pi it is `armv7`, or `aarch64` (if a 64 bit distribution is used).
```
